### DISCUSSION TITLE
Protect position vs initialization

### DISCUSSION THREAD

**Discussion by Robertarduino** [ClassicDIY/SkyeTracker#38] [Q&A]
What is the priotity for movment? ( situation whit hight wind speed) Initialization or Protect? After observation first is Initialization + plus waiting time after Protect position activation after strong wind. 
In real situation its very dangerous because when big wind speed panels request is to go to 0dgr position and after to protect position.

**Comment by graham22** [comment]
Park position is 270° Azimuth, 45° elevation used for maintenance...
Protect position is 270° Azimuth, 90° elevation triggered by the anemometer
Initialization will fully retract both actuators

**Reply by Robertarduino** [reply]
Sorry I mean protect position. When sequence initialization is on going it will be not interupt by ananometer. So it will not stop initialization and switch to protect sequence according to my observation. Its correct?

**Comment by graham22** [comment]
correct, it has to first initialize since the actuators do not provide
position feedback. it will not accept any other commands until the
initialization is completed

On Mon, Jul 11, 2022 at 10:48 AM Robertarduino ***@***.***>
wrote:

> Sorry I mean protect position. When sequence initialization is on going it
> will be not interupt by ananometer. So it will not stop initialization and
> switch to protect sequence according to my observation. Its correct?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ClassicDIY/SkyeTracker/discussions/38#discussioncomment-3122060>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIYATPOVE7ZZ5VHB2WR6FTVTQX2LANCNFSM53G2JLHQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Graham A. Ross
(613) 769-1331

**Reply by Robertarduino** [reply]
From safety point of view.  If signal from anometer is active, better to go at 90dgr and continue to reach East position. After reach continue to 180dgr. It's very dangerous to wait in initialization if strong wind appears. 
What do you think?

**Comment by graham22** [comment]
it only initializes once after a reboot

On Mon, Jul 11, 2022 at 11:31 AM Robertarduino ***@***.***>
wrote:

> From safety point of view. If signal from anometer is active, better to go
> at 90dgr and continue to reach East position. After reach continue to
> 180dgr. It's very dangerous to wait in initialization if strong wind
> appears.
> What do you think?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ClassicDIY/SkyeTracker/discussions/38#discussioncomment-3122459>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIYATJN4DZMIDFJHMLDMUTVTQ43ZANCNFSM53G2JLHQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Graham A. Ross
(613) 769-1331

**Reply by Robertarduino** [reply]
In case of power failure or short ON/OFF during storm it will cause panel in 0dgr.

**Comment by graham22** [comment]
my tracker is powered from my 24V batteries with a buck converter to 12V

I'll be offline (going sailing) for a couple of weeks

On Mon, Jul 11, 2022 at 11:37 AM Robertarduino ***@***.***>
wrote:

> In case of power failure or short ON/OFF during storm it will cause panel
> in 0dgr.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ClassicDIY/SkyeTracker/discussions/38#discussioncomment-3122524>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIYATP3BIC3TY4TI4R7IPTVTQ5SLANCNFSM53G2JLHQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Graham A. Ross
(613) 769-1331
